src/jmcomic/jm_downloader.py

```
@@ -1,12 +1,81 @@
+from functools import wraps
 from typing import NamedTuple
+from time import perf_counter
 
 from .jm_option import *
-from .jm_task_context import bind_jm_task_context
+from .jm_task_context import bind_jm_task_context, get_jm_task_context, jm_task_context
+
+
+def record_download_duration(context_key: str, clock=None):
+    def decorator(func):
+        # 装饰时只解析一次参数名，关键字调用无需在每次执行时重复 inspect。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)